src/pybind/mgr/dashboard/frontend/src/app/core/navigation/notifications/notifications.component.scss

```diff
@@ -7,3 +7,20 @@
 .running:hover i {
   color: bd.$color-solid-white;
 }
+
+a {
+  .dot {
+    background-color: bd.$color-blue;
+    border: 2px solid bd.$color-navbar-bg;
+    border-radius: 50%;
+    height: 11px;
+    position: absolute;
+    right: 17px;
+    top: 10px;
+    width: 10px;
+  }
+
+  &:hover .dot {
+    border-color: bd.$color-primary;
```

Comment:
regarding the contrast. we can change it to white, like we do when there are running tasks:
```suggestion
    background-color: bd.$color-solid-white;
    border-color: bd.$color-primary;
```
![image](https://user-images.githubusercontent.com/399326/87981550-f0c40380-cac4-11ea-9805-5ba64a630890.png)

or maybe change to the previous background color:
```suggestion
    background-color: bd.$color-navbar-bg;
    border-color: bd.$color-primary;
```
![image](https://user-images.githubusercontent.com/399326/87981422-c40fec00-cac4-11ea-8c3b-d7f58c2c6f43.png)


I prefer the white one.